db/models/categoryModels.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../connection');
+
+exports.selectCategories = () => {
+    return db
+        .query(`
+        SELECT * FROM categories;`).then((response)=>{
+           const categories = response.rows;
+           if (!categories) {
```

Comment:
again this bit of code is not needed, the tests you have written are totally fine so please remove this :)